src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
```

Comment:
The package declaration should not be followed by an empty line.

Reply:
I cannot see the empty line.